data/gegno/gegno intro missions.txt

```diff
@@ -432,6 +432,312 @@ mission "Giaru Gegno: Revisit"
+				`	(Leave the planet.)`
+					goto "leave"
+				`	(Prepare for combat.)`
+			`	Engaging your weapon systems, you brace yourself for a hostile encounter as they approach. The Scin ship advances towards yours then slowly comes to a halt in front of you. They haven't fired any weapons, and they aren't displaying any aggressive behavior, so they might not be looking for conflict. Ever so slightly, the warship reverses backwards, then it comes to a stop. You decide to stop your ship as well, wondering what this is about. Once again, the Scin ship reverses, slowly drifting away from you. It appears they want you to follow them.`
+			choice
+				`	(Follow.)`
+					goto "follow"
+				`	(Leave.)`
+					goto "leave"
+			label "leave"
+			action
+				set "gegno: left scin"
+			`	Wary of the Scin warship, you turn around and begin to ascend back into space. The Scin ship advances towards yours, but as it comes closer, it begins to take on an escort formation with you, following you as you leave the planet's atmosphere. Once you finally make it back into space, the Scin ship breaks from formation and flies away. You're unsure of what they're doing on the planet or why you've been lead to them, but it's probably better you be careful not to get involved.`
+				decline
+
+			label "follow"
+			action
+				set "scin: approached epidote"
+			`	You inch your ship forward in acknowledgment. All of a sudden, the second warship on the ground lifts off and forms up behind you, while one in front of you turns around then begins to fly off into the distance. You follow them, curious of where they might be leading you. To your rear, the remaining shuttle takes off, but instead of following you, it heads off into the atmosphere above.`
+			`	Although this planet initially appeared dull due to the several layers of thick clouds, the surface shows occasional signs of life such as plains of grass or occasional dried-up riverbeds, while patches of low fog cover various other locations. As the flight continues, the two Scin ships slowly start to move around so that one is on either side of you. Up ahead in the distance, a large structure comes into view from behind another wall of fog. After getting a bit closer to it, you make out the familiar green colors of the ships flying with you now. The two warships begin to descend towards what you assume to be a sizable Scin freighter sitting in a large open area of sand.`
+			`	Your previous encounter with a Vi warrior was chaotic and unpredictable, so it might be best to be extra cautious in this situation. As you set your ship down near the vessel, the two warships that guided you here also land nearby and power down. Other than the Gegno ships in front of you, there doesn't seem to be any strange activity on your radar at the moment. Environmental readings of the outside surprisingly indicate conditions safe for a human, but it's better to be safe than sorry. You equip your environment suit and your pistol just in case.`
```

Comment:
```suggestion
			`	Your previous encounter with a Vi warrior was chaotic and unpredictable, so you are careful to land slowly and cautiously. As you set the <ship> down near the shuttle, the two warships that guided you here also land nearby and power down. Other than the Gegno ships in front of you, there doesn't seem to be any strange activity on your radar at the moment. Environmental readings of the outside surprisingly indicate conditions safe for a human, but it's better to be safe than sorry. You equip your environment suit and your pistol just in case.`
```

Reply:
This is actually more of an example of telling the player what to think than the above. The text is being suggestive, while your example is telling the player they are careful and intentionally landing slowly and cautiously. I believe the current text to be fine.